src/applications/hca/config/prefill-transformer.js

```diff
@@ -0,0 +1,72 @@
+import { isEqual } from 'lodash';
+import { isInMPI, selectProfile } from 'platform/user/selectors';
+import set from 'platform/utilities/data/set';
+
+/**
+ * Map address object to match the key names in the schema
+ * @param {Array} address - an object with street address data
+ * @returns {Object} - an object of properly-formatted address data
+ */
+export const sanitizeAddress = address => {
```

Comment:
Since this is only used in this file, we could not export it since we don't really need to expose it outside of this file. You would then probably want to make the spec file test this through the `prefillTransformer` method vs the unit test for this function. If we don't plan to expose it outside of this file it maybe makes a little bit of sense to arrange it this way (it's kind of like a private method on a class), but I'm also fine if you'd prefer to leave it as is. Either way, I like the updated test coverage here 👍

Reply:
Yea, good callout here. Little did I realize, I was actually testing the cases for that method already in the transform method itself.